pkg/ccl/importccl/exportcsv_test.go

```diff
@@ -416,10 +432,33 @@ false),(3, 'Carl', 1, 34.214,true),(4, 'Alex', 3, 14.3, NULL), (5, 'Bobby', 2, 3
 				parquet.FieldRepetitionType_REQUIRED,
 				parquet.FieldRepetitionType_OPTIONAL},
 		},
+		{
+			// TODO (mb): switch one of the values in the array to NULL once the
+			// vendor's parquet file reader bug resolves.
+			// https://github.com/fraugster/parquet-go/issues/60
+			//
+			// I already verified that the vendor's parquet writer can write arrays
+			// with null values just fine, so EXPORT PARQUET is bug free; however this
+			// roundtrip test would fail.
+			name: "arrays",
+			prep: []string{"CREATE TABLE atable (i INT PRIMARY KEY, x INT[])",
+				"INSERT INTO atable VALUES (1, ARRAY[1,2]), (2, ARRAY[2]), (3,ARRAY[1,13,5])"},
+			stmt:     `EXPORT INTO PARQUET 'nodelocal://0/arrays' FROM SELECT * FROM atable`,
+			colNames: []string{"i", "x"},
+			vals: [][]interface{}{
+				{int64(1), []interface{}{int64(1), int64(2)}},
+				{int64(2), []interface{}{int64(2)}},
+				{int64(3), []interface{}{int64(1), int64(13), int64(5)}}},
```

Comment:
A couple more cases we might want:

- An empty array, 
- NULL array (as opposed to the array with a null element mentioned in the comment)

Reply:
Done